internal/conf/computed.go

```diff
@@ -1197,6 +1198,8 @@ func defaultMaxPromptTokens(provider conftypes.CompletionsProviderName, model st
 }
 
 func anthropicDefaultMaxPromptTokens(model string) int {
+	//TODO: this doesn't nearly cover all the ways that token size can be specified. https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html
```

Comment:
Ultra-pedantic nit pick around the syntax of a comment.

```suggestion
	// TODO: this doesn't nearly cover all the ways that token size can be specified.
	// See: https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html
```